src/transforms/eigenanalysis.jl

```diff
@@ -36,15 +36,21 @@ function pcaproj(λ, V)
 end
 
 function drsproj(λ, V)
-  Λ = Diagonal(sqrt.(λ))
-  S = V * inv(Λ)
-  S, inv(S)
+  λₛ = sqrt.(λ)
+  Λᵢ = Diagonal(1 ./ λₛ)
```

Comment:
Be consistent. In other parts of the code you use `^-1` to represent inverse matrices, and here you use `i`.